trunk/src/app/srs_app_dash.cpp

```diff
@@ -734,6 +738,14 @@ srs_error_t SrsDash::cycle()
     return err;
 }
 
+srs_utime_t SrsDash::cleanup_delay()
+{
+    if (!enabled) {
+        return 0;
+    }
+    return _srs_config->get_dash_dispose(req->vhost) * 1.1;
```

Comment:
`dash_dispose` is not illustrated in `full.conf`, which can be used as the doc. It's better to add it to `full.conf`.

Another problem, the default `dash_dispose` config is 0, or user can config the `[hls | dash]_dispose` to 0 second, then this `cleanup_delay` will be 0, is the cleanup_delay has to be keep > 0 ? In this case:
`_srs_config->get_dash_dispose(req->vhost) > 0 ? _srs_config->get_dash_dispose(req->vhost) * 1.1 : 1`
Let the `cleanup_delay` at least 1 second.

Reply:
If user config dispose to 0, this cleanup_delay will return 0, which will be ignored.

It works as expectation. Won't fix.

Reply:
I didn't notice the `dash_dispose` default value in `srs_app_config.cpp` already changed to 120 to match the `full.conf`. No problem here.